ecs-cli/modules/cli/local/down_app.go

```diff
@@ -42,8 +42,30 @@ func Down(c *cli.Context) error {
 		network.Teardown(client)
 	}()
 
+	if err := optionsPreCheck(c); err != nil {
+		logrus.Fatalf("Tasks can be either created by local files or remote files")
+	}
+	// if c.Bool(flags.AllFlag) {
```

Comment:
Can we delete commented code? In general commented code creates confusion, noone feels comfortable removing it because you lose context on why it was commented in the first place.

Reply:
Oh my bad. I forgot to delete them